test/src/server/helpers/utils.js

```diff
@@ -0,0 +1,52 @@
+import chai from 'chai';
+import {getNextEnabledAndResultsArray} from '../../../../src/server/helpers/utils';
+
+
+const {expect} = chai;
+
+describe('Testing getNextEnabledAndResultsArray', () => {
+	it('newResultsArray size should be 9 and nextEnabled true when arrayLength=10 and size=9', () => {
```

Comment:
Good unit tests ! You're in the right direction.

I think the description of the tests can be improved: currently you describe your expected conditions instead of the underlying scenario you are testing for. If I'm reading these tests without any knowledge of the code, I won't know what `newResultsArray` or `nextEnabled` are,


I found it help to start the sentence with "it should" followed by the scenario you have in mind.

For example for this test, I would write
`it('should return an array of required length and nextEnabled:true if there are more results than required'…`

It is more descriptive of what you are conceptually testing for.

I'll let you find a more descriptive title for the other tests :)